index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Landing Page</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <style>
+      body {
+        display: flex;
+        flex-direction: column;
+        background-color: #eee;
+      }
+
+      @media screen and (min-width: 200px) {
+        body {
+          background-color: #eee;
+        }
+      }
+    </style>
```

Comment:
I can see that you are using Internal CSS styling. To write clean and reusable code consider using External CSS. You can do this by moving lines 8 - 18 to your style.css and removing it from this file "index.html".

Reply:
The internal CSS cannot be accessed by more than one html pages, it is always better to store CSS in another file. This will not only make your work cleaner but also accessible by other pages on the site. 

>  Kindly take note to always give comment in a polite manner as a suggestion instead of order and always give reason(s) for the suggested changes.

Reply:
Well noted 📝. Thanks for the constructive feedback 🙏🏽